src/command_modules/azure-cli-hdinsight/azure/cli/command_modules/hdinsight/custom.py

```diff
@@ -99,13 +100,13 @@ def create_cluster(cmd, client, cluster_name, resource_group_name, cluster_type,
         raise CLIError('Either the default container or the default filesystem can be specified, but not both.')
 
     # Retrieve primary blob service endpoint
+    is_wasb = True if not storage_account_managed_identity else False
```

Comment:
maybe just `is_wasb = not storage_account_managed_identity`?

Reply:
Thanks for advance! I will update the logic.